src/NIST_800-53/NIST_800-53.ts

```diff
@@ -0,0 +1,64 @@
+/*
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+import { Annotations, CfnResource, IConstruct } from '@aws-cdk/core';
+import { NagPack } from '../common';
+
+import { efs_encrypted_check } from './EFS';
+/*import {
+  dms_replication_not_public
+} from './DMS';*/
+/**
+ * Check Best practices based on NIST 800-53 Operational Best Practices
+ *
+ */
+export class NIST_80053StorageChecks extends NagPack {
```

Comment:
Storage Checks should not be their own class. They should be functions under the existing NIST_800_53 pack